arviz/plots/ppcplot.py

```diff
@@ -1,10 +1,10 @@
 """Posterior predictive plot."""
 import numpy as np
-from .kdeplot import plot_kde
+from .kdeplot import plot_kde, _fast_kde
 from .plot_utils import _scale_text, _create_axes_grid, default_grid
 
 
-def plot_ppc(data, kind='kde', alpha=0.2, mean=True, figsize=None, textsize=None, data_pairs=None):
+def plot_ppc(data, kind='density', alpha=0.2, mean=True, figsize=None, textsize=None, data_pairs=None):
```

Comment:
update the docstring. We should also check if the user is passing valid arguments (density or cumulative)